src/main/java/frc/team2412/robot/commands/launcher/FullTargetCommand.java

```diff
@@ -0,0 +1,73 @@
+package frc.team2412.robot.commands.launcher;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.interpolation.InterpolatingTreeMap;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.team2412.robot.subsystems.DrivebaseSubsystem;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+import frc.team2412.robot.util.LauncherDataLoader;
+import frc.team2412.robot.util.LauncherDataPoint;
+import java.nio.file.FileSystems;
+
+public class FullTargetCommand extends Command {
+
+	private final Pose2d SPEAKER_POSE = new Pose2d(0.0, 5.55, Rotation2d.fromRotations(0));
+
+	private Rotation2d yawAngle;
+	private double angle;
+	private double rpm;
+	private Pose2d robotPose;
+	private Pose2d relativeSpeaker;
+	private double distance;
+	private LauncherDataPoint dataPoint;
+	private Command setAngleCommand;
+	private Command setLaunchSpeedCommand;
+	private Command rotateToAngle;
+	private InterpolatingTreeMap<Double, LauncherDataPoint> launcherData;
+
+	DrivebaseSubsystem drivebaseSubsystem;
+	LauncherSubsystem launcherSubsystem;
+
+	public FullTargetCommand(
+			LauncherSubsystem launcherSubsystem, DrivebaseSubsystem drivebaseSubsystem) {
+		this.launcherSubsystem = launcherSubsystem;
+		this.drivebaseSubsystem = drivebaseSubsystem;
+		setLaunchSpeedCommand = new SetLaunchSpeedCommand(launcherSubsystem, () -> rpm);
+		setAngleCommand = new SetAngleCommand(launcherSubsystem, () -> angle);
+		rotateToAngle = drivebaseSubsystem.rotateToAngle(() -> yawAngle, false);
+		launcherData =
+				LauncherDataLoader.fromCSV(
+						FileSystems.getDefault()
+								.getPath(Filesystem.getDeployDirectory().getPath(), "launcher_data.csv"));
```

Comment:
This is likely slow & noisy (in terms of impact on the Java garbage collector due to the string operations in the parser), so best to do infrequently (ideally just once). I'd recommend making `launcherData` static, so this is only run once when the first instance of this class is created & future instances just use the same data.